### DISCUSSION TITLE
Obtrain the volume fraction of fiber texture component

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#1664] [Ask Anything] [answered]
Hi there,

By mtex, we can get the volume fraction of some standard textures like cube, goss.
The code is :
```matlab
c =  orientation.cube(CS,SS);
volume_c =  volume(odf,c,15*degree);
```
or we can use it for a specific texture like 
```
s = orientation.byMiller([1 2 3],[6 3 -4],CS,SS);
```
My question is how to obtain the volume fraction of a fiber texture, such as <111>||RD
is that `orientation.byMiller([1 1 1],CS,SS);`?
     
Thanks

**Comment by kilir** [comment, marked as answer]
Hi,
try
```
f = fibre(Miller(1,1,1,CS),xvector) % if this is the fibre you'd want
volume(odf,f,15*degree)
```

Cheers,
Rüdiger

**Reply by Brian-Liu-rx** [reply]
Hi, thanks for reply！
The fibre function here is to get the line between two orientations or to get a fibre texture that has the <111> ||  X direction?

**Reply by kilir** [reply]
Hi,
both.
See here: https://mtex-toolbox.github.io/fibre.fibre.html
Cheers,
Rüdiger

**Reply by Brian-Liu-rx** [reply]
Nice！Thanks
